README.md

```diff
@@ -216,50 +251,118 @@ Example usage:
   lines so you'll need to be careful to reconstruct this
 - `rails debug:saml['tmp/your_saml_to_debug.txt']`
 
-## Data Loading
+## User roles
+
+There are a few user roles that provide different levels of permissions. The
+abilities of each role are defined in the [ability model](https://github.com/MITLibraries/thing/blob/master/app/models/ability.rb).
+
+`Basic` is the default user role and is assigned when a user self creates an
+account. Self creation is the only supported way to create an account. If we
+have a new staff member, they first need to login to self create an account
+with the basic role and then admin staff can assign them appropriate roles.
+
+`Thesis Processor` is assigned to any users that process theses.
+
+`Transfer Submitter` is assigned to users that transfers thesis files (typically department admins).
+
+`Thesis Admin` can do everything a `Thesis Processor` can do but can also create
+and update any thesis (not just their own like a `Basic` user).
+
+Roles and the `Admin` flag can be assigned in the Users administrate dashboard in the web UI.
+
+## Audit trail
+
+We use the [`paper_trail`](https://github.com/paper-trail-gem/paper_trail) gem to maintain audit trails of certain
+objects. Classes with `paper_trail` enabled include Thesis, ArchivematicaAccession, and Hold.
 
-There are three types of data that get loaded into this system:
+Following maintainer recommendations, we chose to migrate our `paper_trail` data from YAML to JSON. In YAML, enum fields
+were stored as integers and mapped to their string values as part of the deserialization process. After migrating to
+JSON, new enums are correctly stored as their mapped string values, but legacy data is still stored in the database
+as integers.
 
-### Database Seeds
+After consulting with stakeholders, we decided to leave the legacy data as is and map the enums in the UI,
```

Comment:
I think this note can allow us to close https://mitlibraries.atlassian.net/browse/ETD-631

Reply:
Agreed. That ticket is in review and linked from the commit message. (I'm a bit ambivalent about whether this sort of thing should go into dev docs, but personally I'd look for it in the readme.)